### PR TITLE
[REVIEWS-86] Added and updated API documentation to readme file.

### PR DESCRIPTION
**Pull request by Arturo777**
#### What problem is this solving?

Added and updated API documentation to readme file.

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -154,6 +154,429 @@ In order to apply CSS customizations in this and other blocks, follow the instru
 | `writeReviewContainer`    |
 | `writeReviewButton`       |
 
+# API
+
+## Get Product Rating
+
+
```
Comment (aurorahey) [inline docs/README.md:161]: delete this extra line

**Review by aurorahey — APPROVED** [review]
styles and formating looks good but there are some redundant info in the examples

**Review by btalma — DISMISSED** [review]

**Review by arturmagalhaesjr — APPROVED** [review]

**Review by arturmagalhaesjr — DISMISSED** [review]

**Review by arturmagalhaesjr — APPROVED** [review]
LGTM

File: docs/README.md [hunk 2]
```diff
@@ -154,6 +154,438 @@ In order to apply CSS customizations in this and other blocks, follow the instru
...
+```
+
+## Save Review
+
+![Generic badge](https://img.shields.io/badge/POST-blue.svg) `https://{{accountName}}.{{enviroment}}/reviews-and-ratings/api/review/`
+
+### cURL
+
+```
+curl --location --request POST 'https://arturoreviews--sandboxusdev.myvtex.com/reviews-and-ratings/api/review/' \
```
Comment (polishq) [inline docs/README.md:200]:
```suggestion
curl --location --request POST 'https://{{accountName}}.{{environment}}/reviews-and-ratings/api/review/' \
```

File: docs/README.md [hunk 3]
```diff
@@ -154,6 +154,438 @@ In order to apply CSS customizations in this and other blocks, follow the instru
...
+            "pastReviews": null
+        },
+        {
+            "id": "2",
+            "productId": "880035",
+            "rating": 5,
+            "title": "logged in",
+            "text": "it's cool",
+            "reviewerName": "Brian",
+            "shopperId": "brian.talma@vtex.com.br",
```
Comment (polishq) [inline docs/README.md:399]: Make this a generic email

File: docs/README.md [hunk 4]
```diff
@@ -154,6 +154,438 @@ In order to apply CSS customizations in this and other blocks, follow the instru
...
+```
+
+## Delete Multiple Reviews
+
+![Generic badge](https://img.shields.io/badge/DELETE-red.svg) `https://https://{{accountName}}.{{enviroment}}/reviews-and-ratings/api/reviews/`
+
+### cURL
+
+```
+curl --location --request DELETE 'https://https://{{accountName}}.{{enviroment}}/reviews-and-ratings/api/reviews/' \
```
Comment (polishq) [inline docs/README.md:474]:
```suggestion
curl --location --request DELETE 'https://{{accountName}}.{{enviroment}}/reviews-and-ratings/api/reviews/' \
```

File: docs/README.md [hunk 5]
```diff
@@ -154,6 +154,438 @@ In order to apply CSS customizations in this and other blocks, follow the instru
...
+### Response Body Example
+```
+{
+    "id": "c66d8bc0-787c-11ec-82ac-028dd4526e77",
+    "productId": "880035",
+    "rating": 3,
+    "title": "I am in Korean",
+    "text": "ko-KR",
+    "reviewerName": "ko-KR",
+    "shopperId": "au.roraercc@gmail.com",
```
Comment (polishq) [inline docs/README.md:566]: Replace with a generic email

**Review by polishq — DISMISSED** [review]
LGTM

**Review by arturmagalhaesjr — APPROVED** [review]
LGTM

**Review by karenkrieger — DISMISSED** [review]
Documentation LGTM

**Review by karenkrieger — APPROVED** [review]
LGTM

**Review by arturmagalhaesjr — APPROVED** [review]

**Comment by kaibrockelt** [comment]
Looking good to me.
just one thing: the Review API.pdf does not indicate which versions of the API this document is about. When you have a look at the v3 one, it becomes clear. but when you just stumble upon this doc (somebody might email it) it it not clear what versio of the API it talks about. This could be added.

**Comment by kaibrockelt** [comment]
okay.. we DELETED those files. Lack of experience. (┛◉Д◉)┛彡┻━┻
All good.
i think we shall merge and leave the table in one piece.
┬─┬ノ( º _ ºノ)

**Review by kaibrockelt — COMMENTED** [review]
seems good to me!

**Pull request merged**